### PR TITLE
site: fix image width issue in markdown

### PR DESCRIPTION
**Pull request by Jarryxin**
docs中 Img 图片展示异常

[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE_CN.md?plain=1)

### 🤔 This is a ...

- [ ] 🆕 New feature
- [x] 🐞 Bug fix
- [x] 📝 Site / documentation improvement
- [ ] 📽️ Demo improvement
- [ ] 💄 Component style improvement
- [ ] 🤖 TypeScript definition improvement
- [ ] 📦 Bundle size optimization
- [ ] ⚡️ Performance optimization
- [ ] ⭐️ Feature enhancement
- [ ] 🌐 Internationalization
- [ ] 🛠 Refactoring
- [ ] 🎨 Code style optimization
- [ ] ✅ Test Case
- [ ] 🔀 Branch merge
- [ ] ⏩ Workflow
- [ ] ❓ Other (about what?)

### 🔗 Related Issues
image
文档图片在小屏幕展示异常
> - Describe the source of related requirements, such as links to relevant issue discussions.
> - For example: close #xxxx, fix #xxxx

### 💡 Background and Solution

> - The specific problem to be addressed.
> - List the final API implementation and usage if needed.
> - If there are UI/interaction changes, consider providing screenshots or GIFs.

### 📝 Change Log

> - Describe the impact of the changes on developers, not the solution approach.
> - Reference: https://ant.design/changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    Update the style of images in the document    |
| 🇨🇳 Chinese |   更新文档中图片的样式     |

### CODE REVIEW

File: .dumi/theme/common/styles/Markdown.tsx [hunk 1]
```diff
@@ -37,7 +37,6 @@ const GlobalStyle: React.FC = () => {
         .markdown p > img {
           margin: 34px auto;
           box-shadow: 0 8px 20px rgba(143, 168, 191, 0.35);
-          max-width: 1024px;
```
Comment (li-jia-nan) [inline .dumi/theme/common/styles/Markdown.tsx:40]: 感觉这个不应该去掉，加个 width: 100% 就可以了吧？

**Comment by li-jia-nan** [comment]
另外 PR title 麻烦用英文哈

File: .dumi/theme/common/styles/Markdown.tsx (hunk 1 above)
Comment (Jarryxin) [reply .dumi/theme/common/styles/Markdown.tsx:40]: 如果是小图也100%不合理，第26行有 img 的样式控制
`
 .markdown img {
          max-width: calc(100% - 32px);
          max-height: 100%;
        }
`

**Comment by Jarryxin** [comment]
> 另外 PR title 麻烦用英文哈

改了

File: .dumi/theme/common/styles/Markdown.tsx (hunk 1 above)
Comment (zombieJ) [reply .dumi/theme/common/styles/Markdown.tsx:40]: 试了一下，`width: 100%` 可以的。再看看？

Comment (Jarryxin) [reply .dumi/theme/common/styles/Markdown.tsx:40]: > 试了一下，`width: 100%` 可以的。再看看？

如果是文档中是小图也用 100%不合理。

Comment (zombieJ) [reply .dumi/theme/common/styles/Markdown.tsx:40]: 小图应该是打开一个 Image 会比较好，但是文档内展示 100% 是没问题的

Comment (afc163) [inline .dumi/theme/common/styles/Markdown.tsx:40]: 这句话是 https://github.com/ant-design/ant-design/pull/43707 引入的，移除掉并给对应图片增加默认宽度即可。

File: docs/blog/hydrate-cssinjs.zh-CN.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ author: zombieJ
 
 在 [组件级别的 CSS-in-JS](/docs/blog/css-in-js) 一文中，我们提过。Ant Design 的 Cache 机制并不需要计算出完整的样式。对于组件库而言，只要通过 Token 和 ComponentName 就可以确定生成样式一致性，所以我们可以提前计算出 hash 值：
 
-![Component CSS-in-JS](https://mdn.alipayobjects.com/huamei_7uahnr/afts/img/A*yZMNSYVtxnAAAAAAAAAAAAAADrJ8AQ/original)
+<img alt="CSS-in-JS hash compute way" width="800" src="https://mdn.alipayobjects.com/huamei_7uahnr/afts/img/A*yZMNSYVtxnAAAAAAAAAAAAAADrJ8AQ/original" />
```
Comment (afc163) [inline docs/blog/hydrate-cssinjs.zh-CN.md:39]:
```suggestion
<img alt="CSS-in-JS hash 计算方式" width="800" src="https://mdn.alipayobjects.com/huamei_7uahnr/afts/img/A*yZMNSYVtxnAAAAAAAAAAAAAADrJ8AQ/original" />
```

File: .dumi/theme/common/styles/Markdown.tsx (hunk 1 above)
Comment (afc163) [reply .dumi/theme/common/styles/Markdown.tsx:40]: 这句是 https://github.com/ant-design/ant-design/pull/43707 引入的，移除之并给对应图片增加默认宽度即可。

**Review by afc163 — APPROVED** [review]

**Pull request merged**